samples/samples/autocommit.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2020 Google LLC
+#
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file or at
+# https://developers.google.com/open-source/licenses/bsd
+
+import argparse
+
+from google.cloud.spanner_dbapi import connect
+
+
+def enable_autocommit_mode(instance_id, database_id):
+    """Enables autocommit mode."""
+    # [START enable_autocommit_mode]
```

Comment:
I see this matches spanner-python, the point is to use these samples in embedded docs?

Reply:
Got it, these are docs snippets.